ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs

```diff
@@ -53,6 +54,11 @@ class DevSupportManager : IDevSupportManager, IDisposable
         private Action _dismissDevOptionsDialog;
         private bool _devOptionsDialogOpen;
 
+        private class NoopDisposable : IDisposable
```

Comment:
We already have a dependency on System.Reactive.Core, which has Disposable.Empty